Packages/com.unity.inputsystem/InputSystem/Devices/Keyboard.cs

```diff
@@ -1985,6 +2052,7 @@ public string keyboardLayout
         /// <param name="key">Key code of key control to return.</param>
         /// <exception cref="ArgumentOutOfRangeException">The given <paramref cref="key"/> is not valid.</exception>
         /// <remarks>
+        /// <see cref="KeyControl"/> representing a combined left and right alt key.
```

Comment:
Same as above comment

Reply:
fixed in [8ad97b6](https://github.com/Unity-Technologies/InputSystem/pull/2069/commits/8ad97b6e1a384f347459dbfa06ba72c998464ce4)
I don't know how that slipped in -Thanks